annotation-web/src/main/java/eu/europeana/annotation/web/service/impl/BaseAnnotationValidator.java

```diff
@@ -532,10 +537,11 @@ private void validateSubtitleBody (Body body) throws PropertyValidationException
     	try {
     		subtitleHandler.parseSubtitle(body.getValue(), body.getContentType());
     	}
-    	catch (FileFormatException | IOException e) {
+    	catch (FileFormatException | IOException | ClassNotFoundException | NoSuchMethodException | SecurityException 
```

Comment:
please move the handling of these exceptions to the handler class

Reply:
This is not any more the part of the code, only the exceptions thrown during the direct subtitle validation call.